app/controllers/sessions_controller.rb

```diff
@@ -2,7 +2,7 @@ class SessionsController < ApplicationController
 
   def create
     domain = env["omniauth.auth"]["info"]["email"].split('@').last
-    allowed_domains = ['raybeam.com']
+    allowed_domains = ENV.fetch('allowed_domains').split(',')
```

Comment:
Why is this now `ENV.fetch(...)`? Shouldn't this pull from the YAML file?

Also, since we'll be getting this from a YAML file, we should probably drop the `.split(',')`.